src/resources/2015/Dinseylandfill.ts

```diff
@@ -0,0 +1,316 @@
+import {
+  indexOf,
+  Item,
+  Location,
+  myAdventures,
+  myAscensions,
+  runChoice,
+  visitUrl,
+} from "kolmafia";
+import { getRemainingLiver, have as haveItem } from "../../lib";
+import { get, set } from "../../property";
+import { $item, $location } from "../../template-string";
+
+export function have(): boolean {
+  return get("stenchAirportAlways");
+}
+
+export function available(): boolean {
+  return have() || get("_stenchAirportToday");
+}
+
+class questData {
```

Comment:
Do we want to extract Quest into a class elsewhere and then make this extend it? Or prove it here and see for the future. Also please PascalCase class names

Reply:
If someone else is willing to extract and extend it, I have no issue with that, but I'm happy to prove it here and wait and see if we want to do that when we come to implementing other realms